libnmstate/nm/checkpoint.py

```diff
@@ -112,26 +70,32 @@ def create(self):
         devs = []
         timeout = self._timeout
         cp_flags = (
-            CHECKPOINT_CREATE_FLAG_DELETE_NEW_CONNECTIONS
-            | CHECKPOINT_CREATE_FLAG_DISCONNECT_NEW_DEVICES
+            common.NM.CheckpointCreateFlags.DELETE_NEW_CONNECTIONS
+            | common.NM.CheckpointCreateFlags.DISCONNECT_NEW_DEVICES
         )
+        userdata = self
+
+        self._ctx.register_async("Create checkpoint")
         try:
-            dbuspath = self._manager.interface.CheckpointCreate(
-                devs, timeout, cp_flags
+            self._ctx.client.checkpoint_create(
+                devs,
+                timeout,
+                cp_flags,
+                self._ctx.cancellable,
+                CheckPoint._checkpoint_create_callback,
```

Comment:
Accessing private member is not suggested. Use `self` here.

Reply:
Done!